processor/geoipprocessor/config.go

```diff
@@ -0,0 +1,56 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package geoipprocessor // import "github.com/open-telemetry/opentelemetry-collector-contrib/processor/geoipprocessor"
+
+import (
+	"fmt"
+
+	"github.com/open-telemetry/opentelemetry-collector-contrib/processor/geoipprocessor/internal/conventions"
+	"github.com/open-telemetry/opentelemetry-collector-contrib/processor/geoipprocessor/internal/provider"
+)
+
+// Field represents an attribute used to look for the IP value.
+type Field string
+
+// Config holds the configuration for the GeoIP processor.
+type Config struct {
```

Comment:
Can we remove the `providers`, `metadata` and `fields` from config in this skeleton PR? It would make much more sense to start adding configuration fields together with the actual implementation.

Reply:
Sounds good, let's keep it as a base skeleton. Removed in https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/33201/commits/e4e2f0ebab7081668df988bb9ee3a294622e6612